### PR TITLE
Fixes some misplaced vents and scrubbers

### LINKED ISSUE #5590: Brig Bay has an extra scrubber vent
**Problem Description**:
What is the problem?

Brig Bay has an extra scrubber vent not attached to the scrubber system

http://imgur.com/a/wW4cy - Underneth the body scanner.

### PR DESCRIPTION
**Pull request by monster860**
- Removes an extraneous disconnected scrubber in brig medbay (fixes #5590)
- Removes an extraneous vent and scrubber, and connects up 1 scrubber, in engineering foyer
- Moves a misplaced scrubber in Xeno to it's correct position

### CODE REVIEW

File: _maps/map_files/cyberiad/cyberiad.dmm [hunk 1]
```diff
@@ -9010,6 +9005,13 @@
...
+"vWi" = (/obj/structure/window/reinforced{dir = 8},/obj/structure/table/reinforced,/obj/structure/cable{d1 = 1; d2 = 2; icon_state = "1-2"; tag = ""},/obj/machinery/door_control{id = "xenobio5"; name = "Containment Blast Doors"; pixel_x = 0; pixel_y = 4; req_access_txt = "55"},/turf/simulated/floor/plasteel{dir = 1; icon_state = "warning"},/area/toxins/xenobiology)
+"BCJ" = (/obj/structure/table,/obj/item/weapon/paper_bin{pixel_x = 1; pixel_y = 9},/obj/machinery/atmospherics/pipe/simple/hidden/supply{dir = 4},/turf/simulated/floor/plasteel,/area/engine/break_room)
+"Nyr" = (/obj/item/weapon/storage/firstaid/regular,/obj/item/weapon/storage/firstaid/adv,/obj/item/weapon/reagent_containers/syringe/charcoal,/obj/item/weapon/reagent_containers/glass/bottle/epinephrine,/obj/structure/table/glass,/turf/simulated/floor/plasteel{tag = "icon-whitebluecorner (WEST)"; icon_state = "whitebluecorner"; dir = 8},/area/security/medbay)
+"ROw" = (/obj/machinery/vending/tool,/turf/simulated/floor/plasteel,/area/engine/equipmentstorage)
```
Comment (Fox-McCloud) [inline _maps/map_files/cyberiad/cyberiad.dmm]: Why these are appended like this is a bit suspect.

Comment (monster860) [inline _maps/map_files/cyberiad/cyberiad.dmm]: FastDMM assigns keys randomly.

It has caused no problems that I can see.

Comment (Fox-McCloud) [inline _maps/map_files/cyberiad/cyberiad.dmm]: I'm still not keen on Paradise being the guinea pig for your program.

Comment (monster860) [inline _maps/map_files/cyberiad/cyberiad.dmm]: Paradise is not the guinea pig for my program

FTL13 is, and nothing bad has happened so far from it's use.

Comment (Fox-McCloud) [inline _maps/map_files/cyberiad/cyberiad.dmm]: I'd still prefer it not be used here.

**Comment by Fox-McCloud** [comment]
Going to fix this as requested or?

**Pull request closed without merging**